client/src/app/(main)/message/company/[conversationId]/page.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import { OpportunityStats } from "@/ui/explore/OpportunityStats";
+import { OpportunityStats } from "@/ui/message/OpportunityStats";
 import { Conversation } from "@/ui/message/Conversation";
 export default function Page({
   params,
 }: {
-  params: { conversationId: string };
+  params: { conversationId: string, opportunity_id: string };
 }) {
-  const { conversationId } = params;
+  const { conversationId, opportunity_id } = params;
```

Comment:
The opportunity ID will not be passed in as  url param. So you can just remove this

Reply:
ok